phpcs.xml

```diff
@@ -35,6 +37,12 @@
 		<exclude-pattern>tests/*</exclude-pattern>
 	</rule>
 
-	<rule ref="WordPress-Extra" />
+	<rule ref="WordPress-Extra">
+		<properties>
+			<property name="custom_test_class_whitelist" type="array" value="WP_Ajax_UnitTestCase"/>
```

Comment:
Did not know this was a configuration option within PHPCS.

![Brent Rambo giving a thumbs up](http://i0.kym-cdn.com/entries/icons/original/000/012/982/post-19715-Brent-Rambo-gif-thumbs-up-imgu-L3yP.gif)